product_set/models/product_set_line.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductSetLine(models.Model):
+    _name = "product.set.line"
```

Comment:
This content can be inside `product_set.py` file, as it's a o2m model.

Reply:
In my opinion leaving it in an separate file makes it easier when searching the repository.

Reply:
Personally I  prefer separate files :smirk: